src/main/java/ui/issuepanel/FilterPanel.java

```diff
@@ -50,6 +50,7 @@
     protected Text nameText;
     protected HBox nameBox;
     protected HBox menuBarNameArea;
+    protected HBox menuBarCloseArea;
```

Comment:
menuBarCloseButton? This is expressly meant to be clicked so it could be given a more specific name.